### PR TITLE
POL2943 gabor perf tuneup 3

### PR DESCRIPTION
**Pull request by friskluft**
Changing Gabor feature defaults via Python API

### CODE REVIEW

**Review by sameeul — APPROVED** [review]

File: src/nyx/python/nyxus/nyxus.py [hunk 1]
```diff
@@ -347,6 +356,32 @@ def roi_blacklist_get_summary (self):
...
+        """Sets parameters of feature GABOR
+
+        Keyword args:
+        * kersize (int): size of filter kernel's side. Example: customize_gabor_feature(kersize=16)
+        * gamma (float): aspect ratio of the Gaussian factor. Example: customize_gabor_feature(gamma=0.1)
+        * sig2lam (float): spatial frequency bandwidth. Example: customize_gabor_feature(sig2lam=0.8)
+        * f0 (float): frequency of the baseline lowpass filter as denominator of `\pi`. Example: customize_gabor_feature(f0=0.1)
+        * theta (float): orientation of the Gaussian in degrees 0-180. Example: customize_gabor_feature(theta=1.5708)
+        * thold (float): lower threshold of the filtered image to baseline ratio. Example: customize_gabor_feature(thold=0.025)
+        * freqs (str): comma-separated denominators of `\pi` as frequencies of Gabor filter's harmonic factor. Example: customize_gabor_feature(freqs="1,2,4,8,16,32,64")
```
Comment (sameeul) [inline src/nyx/python/nyxus/nyxus.py:369]: According to the feedback from Hythem, this need to be a List of numbers.

File: src/nyx/python/nyxus/nyxus.py [hunk 2]
```diff
@@ -58,8 +69,7 @@ def __init__(
         n_feature_calc_threads: int = 4,
         n_loader_threads: int = 1,
         using_gpu: int = -1,
-        ibsi: bool = False
-    ):
```
Comment (sameeul) [inline src/nyx/python/nyxus/nyxus.py:62]: my suggestion will be to add something like **gabor_kwargs here.

File: src/nyx/python/nyxus/nyxus.py (hunk 1 above)
Comment (friskluft) [reply src/nyx/python/nyxus/nyxus.py:369]: I have a different view of utility of these string configurations, for example, an ability to add support of directional harmonics freqs="30:1,2,3;60:8,16;120:8,16;150:1,2,3" but OK, I'll rework this into a number list.

File: src/nyx/python/nyxus/nyxus.py [hunk 3]
```diff
@@ -347,6 +356,32 @@ def roi_blacklist_get_summary (self):
             return None
         return s
 
+    def customize_gabor_feature (self, **kwargs):
```
[the remark this answers is not in the document]
Comment (friskluft) [reply src/nyx/python/nyxus/nyxus.py:359]: Customizing feature parameters via the class constructor or single point of initialization is (a) textually awkward when multiple features need to be preset in practical feature extraction cases and (b) incorrect when we need to fit some feature's parameters, for example, doing something like \arg\max_{\gamma,\lambda,\theta} \sum\frac{between}{within} for the Gabor feature - this way we'd need to construct a Nyxus instance in each iteration. 
I suggest to route this design via the product owner (Hythem) and I'll quietly implement the PO's design.

**Pull request merged**